pkg/util/pki/csr.go

```diff
@@ -124,14 +124,28 @@ Outer:
 const defaultOrganization = "cert-manager"
 
 // OrganizationForCertificate will return the Organization to set for the
-// Certificate resource.
+// Certificate resource or if X509Name.Organization is set will return that
 // If an Organization is not specifically set, a default will be used.
 func OrganizationForCertificate(crt *v1alpha2.Certificate) []string {
-	if len(crt.Spec.Organization) == 0 {
+	if len(crt.Spec.Organization) == 0 && len(crt.Spec.X509Name.Organization) == 0 {
 		return []string{defaultOrganization}
 	}
 
-	return crt.Spec.Organization
+	if len(crt.Spec.X509Name.Organization) == 0 {
+		return crt.Spec.Organization
+	}
+
+	return crt.Spec.X509Name.Organization
```

Comment:
Shouldn't need these changes once you remove `certificate.spec.subject.organization` 😄 API versioning will take care of it once we release v1alpha3